CONTACT.md

```diff
@@ -0,0 +1,5 @@
+Welcome to the LiveTL community
+
+Grant desired privacy while offering constructive critique.
+Settle disputes within these means.
+Finding yourselves unable, contact the project team at the [Discord server](https://discord.gg/uJrV3tmthg).
```

Comment:
I like the simplicity of this, but can you add this as a 
```markdown
## TLDR
```
to ``CODE_OF_CONDUCT.md`` so we still have the regular COC. It's nice to have the big document since it is a standard COC that works well in other free and open source software projects.

Reply:
@r2dev2 My point is to avoid it since it sometimes creates awful situations with the power it diverts and conflicts it escalates in secrecy. The 2.0 version is worse yet.
The only thing I would add is "answered by ____ the project maintainer(s)." to have a name to go by.
My little text is recited from memory, and something like it works for Weblate and other projects, which is to say Hosted Weblate doesn't have one. Neither does my group of translators.
I wouldn't have contributed if I had seen it from the Weblate component route I took, but it is listed on this main repo.
Love the effort. One group of translators I am in resorted to doing something similar with closed source, so full marks.